src/components/NewMessageModal.vue

```diff
@@ -286,6 +286,57 @@ export default {
 		handleShow(element) {
 			this.additionalTrapElements.push(element)
 		},
+		async onNewSentMailbox(data, account) {
+			showWarning(t('mail', 'Setting Sent default folder...'))
+			let newSentMailboxId = null
+			const mailboxes = this.mainStore.getMailboxes(data.accountId)
+			const sentMailboxId = mailboxes.find((mailbox) => (mailbox.name === (account.personalNamespace ?? '') + 'Sent') || (mailbox.name === (account.personalNamespace ?? '') + t('mail', 'Sent')))?.databaseId
+			if (sentMailboxId) {
+				try {
+					await this.setSentMailboxAndResend(account, sentMailboxId, data)
+					showSuccess(t('mail', 'Default sent folder set'))
+					this.onSend(data)
+
+				} catch (error) {
+					logger.error('could not set sent mailbox', { error })
+					showError(t('mail', 'Couldn\'t set sent default folder, please try manually before sending a new message'))
+				}
+				return
+
+			}
+			logger.info(`creating ${t('mail', 'Sent')} mailbox`)
+			try {
+				const newSentMailbox = await this.mainStore.createMailbox({ account, name: (account.personalNamespace ?? '') + t('mail', 'Sent'), specialUseAttributes: ['\\Sent'] })
```

Comment:
nit: extract new mailbox name into a variable for code readability and to have the actual mailbox name logged, not just the translation